README.md

```diff
@@ -212,6 +212,15 @@ Other Style Guides
         return atom.value + value;
       },
     };
+    
+    //even better
```

Comment:
For one, this isn’t actually better; but for another, this isn’t using concise method shorthand, which is the purpose of this section.